src/unittest/python/test_blur_manager.py

```diff
@@ -0,0 +1,71 @@
+"""
+----------------------------------------------------------------------
+Authors: Stephan Nell
+----------------------------------------------------------------------
+Unit tests for the Blur Manager
+----------------------------------------------------------------------
+"""
+import pytest
+import cv2
+import os
+from image_preprocessing.blur_manager import BlurManager
+
+TEMPLATE_DIR = "{base_path}/../../main/python/image_preprocessing/templates/".format(
+    base_path=os.path.abspath(os.path.dirname(__file__)))
+
+test_image_colour = cv2.imread(TEMPLATE_DIR + "temp_flag.jpg")
+
+
+def test_thresholding_constructor_incorrect_type():
+    """
+    Test to see if constructor check for invalid type i.e not String type
+    """
+    with pytest.raises(TypeError):
+        BlurManager(1, (1, 1))
+
+
+def test_thresholding_constructor_correct_type():
+    """
+    Test to see if constructor valid type
+    """
+    manager = BlurManager("gaussian", (1, 1))
+    assert manager.blur_type is "gaussian"
```

Comment:
Also test whether the `(1, 1)` tuple is correctly assigned to the correct attribute.